docs/add-ins/addin-tutorial.md

```diff
@@ -1,107 +1,219 @@
+
+1. Locate the `ProviderName` element and replace the default value with your company name.
 
    ```xml
     <ProviderName>Contoso</ProviderName>
     ```
-1. Update the `Description` with a description of the add-in.
+1. Locate the `Description` element, replace the default value with a description of the add-in, and save the file.
 
     ```xml
-    <Description DefaultValue="Allows users to access their Gists on GitHub"/>
+    <Description DefaultValue="Allows users to access their gists on GitHub"/>
    ```
 
-Now we'll change the buttons defined by the add-in. For our add-in, we'll implement two buttons: **Insert Gist** and **Insert Default Gist** on the compose message window. However, the current manifest only adds buttons to the read message window. We'll have to add the message compose command surface extension point.
+#### Test the generated add-in
 
-> [!NOTE]
-> For now we will leave the message read command surface as-is. In future parts of this series, we will revisit the message read scenario.
+Before going any further, let's test the basic add-in that the generator created to confirm that the project is setup correctly. 
 
-Locate the line in the manifest that reads `</DesktopFormFactor>`. Above this line, insert the following XML:
+1. At the command prompt, make sure you are in the root directory of your project, and enter `npm start`. This will start a web server at `https://localhost:3000`.
+
+1. Open either Internet Explorer or Microsoft Edge and navigate to `https://localhost:3000`. If the page loads without any certificate errors, proceed to step 4. If your browser indicates the site's certificate is not trusted, proceed to the next step.
+
+1. Office Web Add-ins should use HTTPS, not HTTP, even when you are developing. If your browser indicates the site's certificate is not trusted, you will need to add the certificate as a trusted certificate. See [Adding Self-Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) for details.
+
+    > [!NOTE]
+    > Chrome (web browser) may continue to indicate the the site's certificate is not trusted, even after you have completed the process described in [Adding Self-Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md). Therefore, you should use either Internet Explorer or Microsoft Edge to verify that the certificate is trusted. 
+
+1. After your browser loads the add-in page without any certificate errors, follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to sideload the **manifest.xml** file that's located in the root directory of the project.
+
+1. In Outlook, open a message and choose the **Display all properties** button. If everything's been setup correctly, the task pane will open and render the add-in's welcome page.
```

Comment:
maybe something like "open an existing message" or "open a message (but not a new one)"

Reply:
Good suggestion; will change this.